### ISSUE TITLE
[BUG] "Wake up" button missing from 1.9.1

### ISSUE THREAD

**Issue by Rahzadan** [streetpea/chiaki-ng#472]
**Describe the bug**

In version 1.9.0 and below, there was a "Wake up" button to wake a sleeping PS5.

<img width="951" alt="image" src="https://github.com/user-attachments/assets/bf90987b-d8b0-463e-b97d-896fd27f87b3">

In version 1.9.1, that button is missing, and the button also does not work anymore:

<img width="1140" alt="image" src="https://github.com/user-attachments/assets/2a4f8373-24cf-4747-9d54-0352eba3d400">

**Debug Log**

I don't believe this is needed.

**To Reproduce**
1. Install version 1.9.0
2. Run Chiaki. "Wake up" button is present
3. Close Chiaki and install 1.9.1, and run it.
4. "Wake up" button is missing.
5. Re-install 1.9.0 and run Chiaki.
6. "Wake up" button returns.

**Expected behavior**
The wake up button to be present and functional.

**Screenshots**
See above

**Desktop (please complete the following information):**
 - OS: Windows
 - Version 11
 - Device Desktop PC
 - chiaki-ng Version 1.9.0 and 1.9.1 

**Additional context**
N/A

**Comment by streetpea** [comment]
@Rahzadan the wake-up button is still available for manual connections. For discovered connections, chiaki-ng automatically wakes up the console and waits for the console to be discovered before connecting. This simplifies the process and reduces the amount of errors when connecting.

**Comment by Rahzadan** [comment]
@streetpea Should the console be woken up as soon as chiaki-ng is launched for discovered consoles? I ask because I still have to manually click it with the mouse / initiate the wake-up for it to wake the console and connect. Previously I could just hit triangle (or Y on an xbox controller) to wake the PS5, even on the descovered console. A minor thing - I just found it inconvenient that it was removed. Maybe I'm doing something wrong?
